### ISSUE TITLE
Firefox does not play VP8 or VP9 recorded demo.js video

### ISSUE THREAD

**Issue by guest271314** [davedoesdev/webm-muxer.js#9]
Firefox 90 warns 
```
Media resource /path/to/camera_9.webm could not be decoded. camera_9.webm
Media resource /path/to/camera_9.webm could not be decoded, error: Error Code: NS_ERROR_DOM_MEDIA_METADATA_ERR (0x806e0006) camera_9.webm
```

**Comment by davedoesdev** [comment]
Also having trouble with another WebM file that Chrome can play: https://github.com/chromium/chromium/blob/master/media/test/data/bear-320x240-opus-av_enc-av.webm

**Comment by davedoesdev** [comment]
I guess that one's because it's encrypted

**Comment by davedoesdev** [comment]
OK this plays: https://github.com/chromium/chromium/blob/master/media/test/data/bear-vp9-opus.webm
Maybe there's a difference in metadata which might give a clue.

**Comment by davedoesdev** [comment]
Bit depth missing maybe

**Comment by davedoesdev** [comment]
No that doesn't seem to help

**Comment by davedoesdev** [comment]
OK it's the audio part that's the problem (video only plays)

**Comment by davedoesdev** [comment]
Might try setting the codec delay
UPDATE: This is zero by default both in the encoding source and firefox.

**Comment by davedoesdev** [comment]
Looks like I'll have to build FF and add extra traces to see where it's complaining

**Comment by davedoesdev** [comment]
Fixed by 9b81ec0b914520d4e3cdbeafb2b3d975178b9141
